### ISSUE TITLE
Coverage test

### ISSUE THREAD

**Issue by vladimirkovacevic** [kilibardajovica/gi-project#5]
On a given coverage on input, how do you test if it is obtained in generated reads?

**Comment by kilibardajovica** [comment]
Number of reads to be generated is calculated so they cover required section of the reference genome, as can be seen in source code.

![Annotation 2020-04-28 220705](https://user-images.githubusercontent.com/61651367/80532604-b272e880-899c-11ea-9f3f-0fe86e8e2729.png)

**Comment by vladimirkovacevic** [comment]
OK, let me rephrase the question. Is there guarantees that given coverage is uniform? Meaning for e.g. given coverage=30, how to avoid having regions with coverage 0 and others much higher coverage?

**Comment by kilibardajovica** [comment]
There are no guarantees that coverage is uniform. Position from which reads are generated are random inside reference genome.

**Comment by vladimirkovacevic** [comment]
@kilibardajovica 
sequencing in real life should provide uniform coverage across all positions in the genome. The simulation of sequencing should do the same thing. Think about it, the point of sequencing is to reconstruct the entire DNA of the individual. If all reads, for example, are from chromosome 1, how would we reconstruct sequences from other chromosomes?

**Comment by kilibardajovica** [comment]
I changed read generation position selection, so reads start from previously uncovered parts of sequence.
![Annotation 2020-05-20 001403](https://user-images.githubusercontent.com/61651367/82384293-dcfe1180-9a2f-11ea-83ae-ee4bbe6d5743.png)
